app/styles/_dashboard.scss

```diff
@@ -187,9 +187,43 @@ header.resource-header {
       display: inline-block;
       margin-left: 6px;
     }
+
+    .reveal-link {
+      margin: 0;
+    }
+
+    &.db-connection-url-value {
+      min-width: 600px;
+    }
```

Comment:
I had to fix the db connection URL width to something really high.  Otherwise, in most cases, when the Reveal link is clicked, the element containing the connection URL will grow and wrap to the next line.  That is jarring, so instead I've hardcoded a large width so it should already be wrapped.